docs/cookbook/displaying_content_with_simple_templates.md

```diff
@@ -48,10 +47,9 @@ At this point all Articles should render using the new template. If you do not s
 
 ## Rendering page elements
 
-In the example above you used the `ez_render_field` Twig function to render the 'body' Field of the content item. Of course each Content item can have multiple fields and you can render them in different ways in the template. Other Twig functions let you access other properties of your content. To see an example, let's extend the template a bit:
+In the example above you used the `ez_render_field` Twig function to render the 'body' Field of the content item. Each Content item can have multiple fields and you can render them in different ways in the template. Other Twig functions let you access other properties of your content. To see an example, let's extend the template a bit:
```

Comment:
For: Other Twig functions let you access other properties of your content.
We could use: Other Twig functions let you access different properties of your content. - to avoid repetition